src/ariadne_wallhug.c

```diff
@@ -1084,10 +1080,10 @@ TbBool is_valid_hug_subtile(MapSubtlCoord stl_x, MapSubtlCoord stl_y, PlayerNumb
     return true;
 }
 
+// Why is it here?
```

Comment:
This digging function follows wallhug right?

Reply:
Ok